______MISC/cluebaseAPI-example.ts

```
@@ -0,0 +1,82 @@
+// example from jeopardy boy used here:
+// https://github.com/GuthrieW/jeopardy-bot/blob/main/src/command-groups/jeopardy/cluebase-api.ts
+
+https://github.com/GuthrieW/jeopardy-bot/blob/main/src/command-groups/jeopardy/cluebase-api.ts
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)